lib/routes/shu/xykd.ts

```diff
@@ -0,0 +1,95 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio'; // cheerio@1.0.0
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+const notice_type = {
+    whxx: { title: '上海大学 - 文化信息', url: 'https://www.shu.edu.cn/xnrc/whxx.htm' },
+    xsbg: { title: '上海大学 - 学术报告', url: 'https://www.shu.edu.cn/xnrc/xsbg.htm' },
+};
+
+export const route: Route = {
+    path: '/xykd/:type?',
+    categories: ['university'],
+    example: '/shu/xykd/xsbg',
+    parameters: { type: '分类，默认为学术公告' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.shu.edu.cn/'],
+            target: '/xykd',
+        },
+    ],
+    name: '官网的校园看点',
+    maintainers: ['GhhG123'],
+    handler,
+    url: 'www.shu.edu.cn/',
+    description: `| 文化信息 | 学术报告 |
+  | -------- | --------- |
+  | whxx     | xsbg      |`,
+};
+
+async function handler(ctx) {
+    const type = ctx.req.param('type') ?? 'xsbg';
+    const rootUrl = 'https://www.shu.edu.cn';
+
+    // 发起 HTTP GET 请求
+    const response = await got({
+        method: 'get',
+
+        /* headers: {
+            'user-agent': UA,
+            cookie: await getCookie(ctx),
+        }, */
+        url: notice_type[type].url,
+    });
+
+    const $ = load(response.data);
+
+    const list = $('div.xsbg_list ul li') // 定位到HTML结构中的li元素
+        .map((_, el) => {
+            const item = $(el); // 使用Cheerio包装每个li元素
+            const rawLink = item.find('a').attr('href');
+            const dateParts = item
+                .find('div.sj p')
+                .map((_, p) => $(p).text().trim())
+                .get(); // 提取日期部分
+
+            return {
+                title: item.find('p.bt').text().trim(), // 获取标题
+                link: rawLink ? new URL(rawLink, rootUrl).href : rootUrl, // 生成完整链接
+                pubDate: timezone(parseDate(`${dateParts[1]}-${dateParts[0]}`, 'MM-DD'), +8), // 拼接并解析日期
+                description: item.find('div.zy').text().trim(), // 提取简要描述
+            };
+        })
+        .get();
+
+    const items = await Promise.all(
+        list.map((item) =>
+            cache.tryGet(item.link, async () => {
+                const detailResponse = await got(item.link); // 获取详情页内容
```

Comment:
Use either https://github.com/DIYgod/RSSHub/blob/8b48453ed55165a9e2adbda0705bb8bf6accd1e1/lib/routes/shu/xykd.ts#L46-L54 or https://github.com/DIYgod/RSSHub/blob/8b48453ed55165a9e2adbda0705bb8bf6accd1e1/lib/routes/shu/xykd.ts#L79 but not both style